src/components/ProjectList/listOfProjects.js

```diff
@@ -1,12 +1,18 @@
 const projectList = [
+   {
+    name: 'Ancient Beast',
+    imageSrc: 'https://raw.githubusercontent.com/FreezingMoon/AncientBeast-Website/master/favicon.png',
+    projectLink: 'https://github.com/FreezingMoon/AncientBeast',
```

Comment:
For other projects, we take users to `/contribute` path so they can see issues labeled `good-first-issue` or `help wanted`. For AncientBeast, I think the similar thing would be to point to issues with label `easy`

Reply:
I've noticed that for some of the other projects. Was thinking if to do that as well, though directly linking to the repo makes the readme more exposed and the contribution guide. Either way is fine with me though 👍🏻